CMake/amiga/ppc-warpos.cmake

```
@@ -0,0 +1,37 @@
+set(CMAKE_SYSTEM_NAME Generic)
+set(CMAKE_SYSTEM_PROCESSOR ppc)
+
+set(CMAKE_SYSROOT /gg/ppc-warpos)
```

In order for this to compile on other systems that MorphOS, may I suggest adding the WarpOS prefix path as an option?
```suggestion
option(WARPOS_PREFIX "Path to WarpOS prefix" "/gg")
set(CMAKE_SYSROOT "${WARPOS_PREFIX}/ppc-warpos")
set(CMAKE_FIND_ROOT_PATH "${WARPOS_PREFIX}/ppc-warpos")
set(tools "${WARPOS_PREFIX}/bin/")
```

---

Nowadays only newer gcc/g++ for warpos exist through mos2wos (newlib). But no harm with suggestion if this option works in cmake MOS.